conf/gunicorn-websocket-monolithic.conf.py

```diff
@@ -1,5 +1,8 @@
+from os import environ
+
+
 bind = "unix:/tmp/gunicorn-websocket.sock"
 worker_class = "h.streamer.Worker"
 graceful_timeout = 0
-workers = 2
+workers = environ["WEBSOCKET_NUM_WORKERS"]
```

Comment:
This will now crash if the envvar is missing, which will be easier to debug than it silently defaulting to one worker